solidity/random-beacon/test/system/e2e.test.ts

```diff
@@ -60,44 +68,66 @@ describe("System -- e2e", () => {
   const relayEntrySoftTimeout = 20
   const callbackGasLimit = 56000
   const groupCreationFrequency = 5
-  const groupLifetime = 403200
+  const groupLifetime = 11520 // 2 days assuming 15s block time
```

Comment:
Can we also explain where the value of 2 days come from?

Reply:
I don't remember exactly the history behind these values, but I believe that back then the values or the code was different. Now, we only need to adjust the group creation frequency from 2 -> 5 just because of the existing testing data that was prepared specifically for this frequency. It is also possible to use everything from the `RandomBeacon` but creating a new testing data would take some time and I don't think it should be done here. The rest of the init params are removed now. [39cfc61](https://github.com/keep-network/keep-core/pull/3018/commits/39cfc613e4fe9894160cb40ac4c884623da8259c)